pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -3,6 +3,20 @@ var bsTable;
 $(function() {
 
     bsTable = $('#usersTable').bootstrapTable({
+        buttons: {
+            studentsOnly: {
```

Comment:
Can we add hover text to say what the button does, like the current Refresh and Column buttons have? (the Download button is also missing hover text)

Reply:
Done.